Sources/MapboxCoreNavigation/Distance/LineDistanceInfo.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import MapboxNavigationNative
+
+/**
+ * Contains information about distance to the road object represented as line.
+ */
+public struct LineDistanceInfo {
+
+    /**
+     * Distance along the road graph from current position to entry point of object in meters,
+     * if already "within" object will be equal to zero
+     */
+    public let distanceToEntry: CLLocationDistance
+
+    /** Distance along the road graph from current position to exit point of object in meters. */
+    public let distanceToExit: CLLocationDistance
+
+    /** Distance along the road graph from current position to end of road object */
+    public let distanceToEnd: CLLocationDistance
+
+    /** If we enter road object from it's start, if already "within" object - always false */
+    public let isEntryFromStart: Bool
+
+    /** Length of "long" objects */
+    public let length: Double
```

Comment:
We should document what happens for “short” objects. Is this property set to 0 or some other magic value?

/cc @SiarheiFedartsou

Reply:
Hm, for `LineDistanceInfo` is should be always present. So, I think we should change comment to smth like `length of object in meters`

Reply:
Fixed as part of #2991.